src/main/utils/download.ts

```diff
@@ -46,6 +78,21 @@ export function downloadFileWithProgress(
         progress: 100,
         localUserPath: '',
         isFinished: true,
+        processingSize: 0,
+      },
+    });
+  });
+
+  req.on('error', () => {
+    console.log('download failed');
+    eventsInstance.reply({
```

Comment:
- tutaj najważniejsze jest wysłanie do dashbaordu jak sentry
- zwracając błąd do renderera nie robiłbym w kontekście Procesów jeśli zostaną w takiej formie, bo errory mogą być również później np. czy autentykacji czy avatach itd - dobrze mieć 1 sposób na obsługę wszystkich errorów miedzę mainem a rendererem. Dodatkowo dobrze zwrócić error message do renderera, co ułatwi debugowanie zpakowanych wersji aplikacji. Ten message poczatkowo można console.logować w rendererze

Reply:
chodzi o to, by przesłać informacje o błędzie do renderera